azure-pipelines.yml

```diff
@@ -0,0 +1,98 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+#trigger on master pull
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - 'FakebookNotifications'
+pool:
+  vmImage: 'ubuntu-latest'
+
+stages:
+  - stage: build
+    jobs:
+      - job: build
+      # use an earlier version of .net because sonar cloud doesn't like .net 5
+        variables:
+          sdkVersion: 3.1.x
+          buildConfiguration: 'Release'
+      # build using dotnet 5 
+        steps:
+        - task: UseDotNet@2
+          displayName: dotnet sdk 2.x
+          inputs:
+            packageType: 'sdk'
+            version: '5.x'
```

Comment:
inconsistent display, config, and comments. also a problem in the job lower down.